ops/terraform/main.tf

```diff
@@ -98,6 +98,10 @@ module "ocr_autoscale" {
   weekend_capacity_instances = 1
 }
 
+module "vault" {
+  location       = data.azurerm_resource_group.rg.location
```

Comment:
We should pre-create/manually create vault secrets ahead and bringing them in as data sources similar to how we are with the Resource Group's, like this for example: https://github.com/CDCgov/prime-simplereport/blob/593c56c62847fbf398a0a8cddc5e4069a1499bd1/ops/prod/_data.tf#L50 This is also how CDC azure access will be. 

Unfortunately it looks like we might not have permissions to create key vault secrets at the moment in Azure. I ping'd Josh earlier so see if he can add a role to allow it.

Reply:
We met with Josh Dorothy on Tuesday, November 26 and received permissions.  In that meeting you agreed on this architectural design.  In that meeting, I re-iterated the manual creation of the secret may not be the best practice or most secure.